### DISCUSSION TITLE
Wyze-Bridge + Frigate + go2RTC

### DISCUSSION THREAD

**Discussion by jeeftor** [blakeblackshear/frigate#14017] [Ask A Question]
I have a few old Wyze cams which I'm using with wyze-bridge.

Right now I'm doing:

Reolink -> go2rct -> Frigate
Wyze-Bridge -> Frigate

I was thinking I should also route the wizen cameras through go2rtc? 

Is there a pro/con on this?

**Comment by NickM-27** [comment]
if you want high quality live views then go2rtc must be used. If the jsmpeg view without audio / lower fps works for you then there is no need

**Reply by jeeftor** [reply]
What do you mean? Doesn't wyze-bridge push out RTSP streams regardless?

I got go2rtc configured to use the wyze-bridge streams... but i'm still usign the direct streams in frigate. I guess i could also configure go2rtc stream and compare.

**Reply by NickM-27** [reply]
https://docs.frigate.video/configuration/live
